data/en/ukis_0001.json

```diff
@@ -4144,7 +4144,13 @@
                                 ]
                             },
                             "description": "Estimates are acceptable and percentages must <strong>total 100%</strong>",
-                            "type": "General",
+                            "type": "Calculated",
+                            "calculations": [{
+                                "calculation_type": "sum",
+                                "value": 100,
+                                "answers_to_calculate": ["answer3207", "answer3208", "answer3209", "answer3211"],
+                                "conditions": ["equals"]
+                            }],
```

Comment:
You could previously leave this answer blank (in fact it only validated that the total was less than 100). If no answer is still a requirement then this should actually be:

```
"calculations": [{
    "calculation_type": "sum",
    "value": 0,
    "answers_to_calculate": ["answer3207", "answer3208", "answer3209", "answer3211"],
    "conditions": ["equals"]
},{
    "calculation_type": "sum",
    "value": 100,
    "answers_to_calculate": ["answer3207", "answer3208", "answer3209", "answer3211"],
    "conditions": ["equals"]
}],
```